fusor-ember-cli/app/serializers/deployment.js

```diff
@@ -1,11 +0,0 @@
-import DS from 'ember-data';
-import { ActiveModelSerializer } from 'active-model-adapter';
```

Comment:
no need for separate deployment serializer since it will inherit from serializers/application.js and be of the form DS.JSONAPISerializer